### PR TITLE
2.0 dataset locations

### PR DESCRIPTION
**Pull request by NMPowell99**
Set up remote dataset dag and graph data dataset dag.
The graph data dag is in a working state, but only for NER, keyphrase, RE*, and translation data. 

What still needs to be done:
CRR data: working on that now
POS data: have to rebase java POS changes on batching changes to get that info in the graph, adding it to the training file afterwards should be super simple
sentiment data: need the model node in the graph to know the type of sentiment before adding it to the training file
*RE: add indexes of mentions in sentence (this is optional, RE training will still work without it)

The dockerfile is set to build an image using the dataset dag files. Do we want one dockerfile that the user would change for whichever of the three dags they want, or should we have three separate dockerfiles, and the user can choose one rather than having to change it themselves?

### CODE REVIEW

**Review by alexiudice — APPROVED** [review]
Overviewed and looks good for the changes. 

We'll need to make some changes to the `run_ongdb_query` method that statically connects to an ONgDB instance, but that can come in a future PR.

File: airflow_dag_samples/graph_data_dag/sdk_calls.py [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+                               autopromote, success_handler, failed_handler)
+
+
+def main():
+    fire.Fire(Pipeline)
+
+
+if __name__ == '__main__':
+    main()
+
```
Comment (HarryDunham) [inline airflow_dag_samples/graph_data_dag/sdk_calls.py]: Noting that we'll probably want to clean this up and remove all these extraneous newlines @alexiudice

File: airflow_dag_samples/remote_dataset_dag/example_dag.py [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+dag = DAG('train_model_with_sdk',
+          default_args=default_args,
+          description='train a model using sdk calls',
+          schedule_interval="@daily",
+          catchup=False,
+          user_defined_macros=dict()
+          )
+
+
+def create_volume() -> str:
```
Comment (HarryDunham) [inline airflow_dag_samples/remote_dataset_dag/example_dag.py:45]: I think the below methods are identical to the other example dags volume creation methods. Does it make sense to eventually consolidate these to some util file and import them into each example dag? @alexiudice

**Review by HarryDunham — APPROVED** [review]
Looks pretty good, left a couple comments for us to consider after we get this in.

File: airflow_dag_samples/remote_dataset_dag/example_dag.py (hunk 2 above)
Comment (alexiudice) [reply airflow_dag_samples/remote_dataset_dag/example_dag.py:45]: That definitely makes sense here.

**Pull request merged**